Sources/SwiftlyCore/HTTPClient.swift

```diff
@@ -22,6 +22,100 @@ private func makeRequest(url: String) -> HTTPClientRequest {
     return request
 }
 
```

Comment:
Is it worthwhile breaking out the swift.org data and parsing into a separate file?

Reply:
I'm on the fence about this.

It feels like the swift.org metadata is a foundational use of the swiftly http client, so why not just put the code there? On the other hand, because of the GitHub fallback there's some asymmetry now because it's in another source file. But, there's also a feeling that the fallback may go away someday, so that could be why it is separate.

Reply:
As I see it there is the client and the usage of the client which are two different things